docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -64,6 +64,147 @@ The top bar of Teleport Connect consists of:
 The **status bar** at the bottom displays **cluster breadcrumbs** in the bottom left, indicating
 which cluster the current tab is bound to, and the **Share Feedback** button in the bottom right.
 
+## Connect My Computer
```

Comment:
Is this section big enough that it warrants having a separate page, perhaps written as a how-to guide?

I'm not sure about the how-to guide part, as there's not a lot of explaining what to do.

Reply:
I think this page is starting to burst at the seams a little bit, but we can leave the reorganization for a separate project!